shared/src/commonMain/kotlin/com/br/kmmdemo/ui/shared/KMMDemoSlider.kt

```diff
@@ -16,19 +16,21 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.clip
 import androidx.compose.ui.graphics.Brush
 import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.Dp
 import com.br.kmmdemo.theme.Dimens
 import com.br.kmmdemo.theme.Gradients
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun KMMDemoSlider(
+    topPadding: Dp = Dimens.grid_2,
     initValue: Float,
     valueRange: ClosedFloatingPointRange<Float>,
     onValueChange: (Float) -> Unit = {},
     ) {
     Box(
         modifier = Modifier
-            .padding(vertical = Dimens.grid_2)
+            .padding(vertical = topPadding)
```

Comment:
A cleaner way of doing this would be to pass in a modifier:

fun KMMDemoSlider(....., modifier = Modifier.padding(Dimens.grid_2))

then

Box(modifier = modifier (note the little M).clip(.....)......